src/lib.rs

```diff
@@ -99,22 +99,3 @@ macro_rules! eager_binding {
     ($context_name:tt = $($macro_name:tt!()),*) => {eager! { lazy! { wgpu_macros::generic_bindings! { $context_name = eager!{ $($macro_name!()),*}}}}}
 }
 
-// hi
-/* pub mod bind;
-
-#[macro_use]
-extern crate eager;
-
-my_shader! {compute = {
-    [[buffer loop in out] uint[]] indices;
-    [[buffer in] uint[]] indices2;
-    //[[buffer out] uint[]] result;
-    //[... uint] xindex;
-    {{
-        void main() {
-            // uint xindex = gl_GlobalInvocationID.x;
-            uint index = gl_GlobalInvocationID.x;
-            indices[index] = indices[index]+indices2[index];
-        }
-    }}
-}} */
```

Comment:
I believe this is old stuff no longer relevant, yeah?

Reply:
This appears to be resolved by my latest update